src/auth/FileTokenStorage.ts

```diff
@@ -15,46 +15,37 @@ export class FileTokenStorage implements TokenStorage {
   constructor(private filePath: string) {
   }
 
-  public get(): Promise<string> {
-    return new Promise<string>((resolve: (connectionInfo: string) => void, reject: (error: any) => void): void => {
-      if (!fs.existsSync(this.filePath)) {
-        reject('File not found');
+  public async get(): Promise<string> {
```

Comment:
I don't think removing the promises in this file is the right thing to do. It expects promises are returned, while that's not really the case now

I think the promises were added in all functions here to make it consistent. Because some functions are returning or throwing in a callback, awaiting a `new Promise` seems to be the correct way

Reply:
Martin, I kinda agree, but I'm not sure why it should be a promise in the first place, as no async action is being executed in the function. At the moments. Maybe it could just be a `get` method that returns a string? All actions in the method are currently sync.

Theoretically, the set/remove methods could also become a regular method, without any async or promises, but the error handling is done in the fs.writeFile command here. But that can be changed. 

If you want me to revert the changes, I will do this obviously.

Reply:
The reason that it is a promise is because the write function is using a callback. We only want to return as soon as the callback is called, to make sure that no error is thrown. Hence a promise is used to make it easier, to work with the old callback code and return it promise-like.
And as I said, I think the get function is just made promise-like for it to be consistent across the board.

Reply:
I'll change it back 😊 Will do this later!